plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotIntegrationSmokeTest.java

```diff
@@ -93,7 +94,7 @@ protected QueryRunner createQueryRunner()
     {
         TestingKafka kafka = closeAfterClass(TestingKafka.createWithSchemaRegistry());
         kafka.start();
-        TestingPinotCluster pinot = closeAfterClass(new TestingPinotCluster(kafka.getNetwork()));
+        TestingPinotCluster pinot = closeAfterClass(new TestingPinotCluster(kafka.getNetwork(), true));
```

Comment:
Let me know whether is worth keeping both secured and unsecured ITs

Reply:
I've done this refactoring in the latest commit

Reply:
The overhead is only 2min. Let's keep both tests.